RecoEgamma/EgammaElectronProducers/plugins/TrackingRegionsFromSuperClustersProducer.h

```diff
@@ -181,6 +195,11 @@ fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   desc.add<double>("deltaPhiRegion",0.4);
   desc.add<double>("deltaEtaRegion",0.1);
   desc.add<bool>("useZInVertex", false);
+  desc.add<bool>("useZInBeamspot", true);
```

Comment:
You could add a comment here explaining that "useZInVertex" and "useZInVertex" cannot be both true, see https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Adding_Comments